scripts/exglobal_atmos_pmgr.sh

```diff
@@ -50,7 +50,10 @@ do
       if [ $fhr -eq 0 ]
       then 
 ####        ecflow_client --event release_${RUN}_postanl
-        ecflow_client --event release_postanl
+        if [ -s ${COMIN}/${RUN}.${cycle}.loganl.txt ]
+        then
+          ecflow_client --event release_postanl
+        fi
       fi    
```

Comment:
Maybe compress this if-block to become:
```
if [ $fhr -eq 0 -a -s ${COMIN}/${RUN}.${cycle}.loganl.txt ]
then
   ecflow_client --event release_postanl
fi
```
Unless there are plans to add additional cases/if-statements to the `if [ $fhr -eq 0 ]` block.

Reply:
@KateFriedman-NOAA Good suggestion! I have been waiting for @lgannoaa comments and assume the changes were tested by Lin in ecflow.